### PR TITLE
DOCS-9182: DBM RDS onboarding workflow

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

DOCS-9182. Docs for new DBM RDS onboarding workflow

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

**Review by ynnadkrap — APPROVED** [review]

File: content/en/database_monitoring/setup_rds_quick_install.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+further_reading:
+- link: "/database_monitoring/setup_postgres/"
+  tag: "Documentation"
+  text: "Setting up Postgres"
+- link: "/database_monitoring/setup_postgres/rds"
+  tag: "Documentation"
+  text: "Setting Up Database Monitoring for Amazon RDS managed Postgres"
+---
+
+Database Monitoring Quick Install for RDS enables you to quickly set up Agents to monitor your RDS Postgres instances. After specifying a few options, Datadog generates a CloudFormation template that configures your instance for monitoring and uses AWS ECS to deploy the Agent with recommended DBM configurations to the RDS instance.
```
Comment (janine-c) [inline content/en/database_monitoring/setup_rds_quick_install.md:12]:
```suggestion
Database Monitoring Quick Install for RDS enables you to quickly set up Agents to monitor your RDS Postgres instances. After you specify a few options, Datadog generates a CloudFormation template that configures your instance for monitoring, and uses Amazon ECS to deploy the Agent to the RDS instance with recommended DBM configurations.
```

**Review by janine-c — COMMENTED** [review]
Looks good! Just a few readability edits (and I _think_ addressing the reason behind Vale not liking `AWS ECS`?), but nothing major 🙂

**Review by janine-c — APPROVED** [review]
:shipit:

**Review by renteria1010 — APPROVED** [review]

**Pull request merged**